### ISSUE TITLE
Let users hide boosts from people they follow

### ISSUE THREAD

**Issue by mxamber** [bookwyrm-social/bookwyrm#2757]
I just followed my main account from my bookwyrm account, and my feed is full of stuff I boost on my other account. I'd like to filter that, as is standard on most fediverse software. A "hide boosts from user" button would be greatly appreciated.

**Comment by mouse-reeve** [comment]
Agreed! Standard fediverse content isn't supposed to be displayed in bookwyrm anyway and I haven't figured out why it's getting in. But boosts should be disable-able regardless.
